### PR TITLE
Subterminal precategories and constant functors

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Based on the review of #896.

### CODE REVIEW

File: src/category-theory/constant-functors.lagda.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Constant functors
```
Comment (fredrik-bakke) [inline src/category-theory/constant-functors.lagda.md:1]: I went with a slightly unconventional structure and put all the different notions of constant functors in one file.

Comment (fredrik-bakke) [reply src/category-theory/constant-functors.lagda.md:1]: Note that this is actually similar to how some other files are organized, like `terminal-category`.

File: src/category-theory/indiscrete-precategories.lagda.md [hunk 2]
```diff
@@ -44,7 +45,8 @@ hom-[sets](foundation-core.sets.md) are
 
 This construction demonstrates one essential aspect about precategories: While
 it displays `obj-Precategory` as a [retraction](foundation-core.retractions.md),
-up to weak categorical equivalence, every indiscrete precategory is subterminal.
+every indiscrete precategory is
+[subterminal](category-theory.subterminal-precategories.lagda.md).
```
Comment (fredrik-bakke) [inline src/category-theory/indiscrete-precategories.lagda.md:49]:
```suggestion
[subterminal](category-theory.subterminal-precategories.md).
```

File: src/category-theory/constant-functors.lagda.md (hunk 1 above)
Comment (EgbertRijke) [reply src/category-theory/constant-functors.lagda.md:1]: Very nice, I like it. I think it makes good sense with such simple definitions. Although I can see the file becoming more complicated when we add predicates for `is-constant-functor-Categrory` etc (not asking you to do this).

**Comment by EgbertRijke** [comment]
Beautiful!

**Pull request merged**